src/components/ShareStructure/ShareStructures.vue

```diff
@@ -8,21 +8,22 @@
     />
 
     <share-structure
-      :is-edit-mode="isEditMode"
+      :isEditMode="isEditMode"
       :isCorrection="isCorrectionView()"
-      :incorporation-application="getOriginalIA"
-      :share-classes="getShareClasses"
-      :business-snapshot="getOriginalSnapshot"
-      :resolution-required="resolutionsRequired"
-      :edit-label="editLabel"
-      :edited-label="editedLabel"
+      :incorporationApplication="getOriginalIA"
+      :shareClasses="getShareClasses"
+      :originalShareStructure="originalShareStructure"
```

Comment:
This is the big change. I'm passing in only what share-structure needs to know about. Share-structure does not need to (and should not) know the details of the business snapshot.